### PR TITLE
Changed wording in README

### PR DESCRIPTION
**Pull request by cadenchabot**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,8 @@
 ## Technology
```
Comment (Arvind-Maan) [inline README.md:5]: put technology under the description so we see description first

**Review by Arvind-Maan — APPROVED** [review]
LGTM!

**Review by RikiZimbakov — APPROVED** [review]
job well done.

**Review by andy-tan7 — APPROVED** [review]
Looks good

**Pull request merged**